### ISSUE TITLE
Too few positional arguments: 6 required, 5 given.

### ISSUE THREAD

**Issue by elixiao** [memspace/zefyr#124]
```sh
$ flutter run -d 'iPhone Xʀ'
Launching lib/main.dart on iPhone Xʀ in debug mode...

Compiler message:
file:///program/flutter/.pub-cache/git/zefyr-077cd10dfb04d05ae97801b49160f7af0829ca47/packages/zefyr/lib/src/widgets/selection.dart:509:49: Error: Too few
positional arguments: 6 required, 5 given.
    final toolbar = widget.controls.buildToolbar(
                                                ^
Compiler failed on /Volumes/project/lib/main.dart
Error launching application on iPhone Xʀ.
```

```sh
$flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, v1.7.8+hotfix.2, on Mac OS X 10.14.5 18F132, locale zh-Hans-CN)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
[✓] Xcode - develop for iOS and macOS (Xcode 10.2.1)
[✓] iOS tools - develop for iOS devices
[✓] Android Studio (version 3.4)
[✓] VS Code (version 1.36.0)
[✓] Connected device (1 available)

• No issues found!
```

**Comment by xiongdi9456** [comment]
Tested on Android, the same error

**Comment by w4mxl** [comment]
Got same error here.

**Comment by Luppakorva** [comment]
I rolled back to 1.5.x flutter version for now. Seems to be the only option.

**Comment by w4mxl** [comment]
In order to build the app normally, I temporarily modify the selection.dart:509 into the following:
```
final toolbar = widget.controls.buildToolbar(
        context, editingRegion, 1, midpoint, endpoints, widget.delegate);
```
It’s now working.Waiting for official author repair.

**Comment by pulyaevskiy** [comment]
This is fixed in 0.6.0, please upgrade.
